### DISCUSSION TITLE
sync files across LAN

### DISCUSSION THREAD

**Discussion by edmundlaugasson** [elementary/triage#401] [Ideas]
Would propose to add an app, that enables to sync files across internal network (LAN). In Linux Mint is the app [Warpinator](https://github.com/linuxmint/warpinator) for such purpose. As Linux Mint is based on Ubuntu LTS like Elementary OS, then it would be an easy way to implement same app also for Elementary OS.

**Comment by idottertcrosser** [comment]
I use SyncThing
